fmt.sh

```diff
@@ -129,6 +129,13 @@ echo "\
 ──██────▐█▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓█▌
 "
 
+# In 'test' mode, this checks that the 'yarn.lock' file doesn't need any changes.
+# In 'format' mode, the 'yarn.lock' file will be updated ("formatted") by
+# 'yarn install' below, if needed.
+if [[ $is_test = 1 ]]; then
+  run pre-commit run yarn-lock-check
+fi
```

Comment:
Does this take a long time/use a lot of space+network?

Reply:
not really, and not more than `run yarn install --silent` a few lines down

Reply:
Alright, looks good then :)